java/api/src/main/java/io/ray/api/Ray.java

```diff
@@ -17,6 +17,7 @@
 public final class Ray extends RayCall {
 
   private static RayRuntime runtime = null;
+  public static final String DEFAULT_PLACEMENT_GROUP_NAME = "unnamed_group";
```

Comment:
Better to move it into `AbstractRayRuntime`.

Reply:
updated.